src/locales/fr.json

```
@@ -333,6 +333,26 @@
                 "Atomic": "Bug corrigé dans le tableau périodique",
                 "Gutter": "Gouttière corrigée dans l'application python"
             }
+            "1-0-1": {
+                "XNT" : "XNT Cyclique (Plusieurs appuis sur XNT changent le caractère inséré entre X, N, T et θ)",
+                "NewPol" : "Police changée",
+                "TxtItal" : "Texte en italique dans certains endroits tels que les définitions de fonctions dans l'application fonctions et les textes et commentatires dans l'éditeur Python",
+                "TxtTool" : "Texte plus gros dans les boites à outils avec défilement automatique quand le texte est trop long",
+                "LastApp" : "Raccourci Shift + Ans pour aller à la dernière application utilisée",
+                "CorrPix" : "Correction du pixel décalé sur certaines icônes dans certains thèmes",
+                "Dim" : "Dimension dans les résultats additionnels quand une unité est utilisée",
+                "PyMaJ" : "Mise à jour de MicroPython de 1.17 vers 1.19.1",
+                "PyCrash" : "Correction du crash lors de l'ouverture des scripts",
+                "PyItal" : "Texte en italique dans les chaines de caractères",
+                "AddUlab" : "Ajout de `ulab.utils`",
+                "PyLightControl" : "Contôle de la lumisosité via `set_brightness` et `get_brightness`",
+                "AddStat" : "Moyenne géométrique, Moyenne harmonique et Mode dans l'onglet Stats",
+                "AddFunctionLiseuse" : "Fonctions en plus en URT (Sommes, Produits, Intégrales et coefficients binomiaux)",
+                "AddCaracURT" : "Caractères en plus en URT (`\f` et `\i`)",
+                "CorrBrownColor" : "Correction de la couleur marron en URT",
+                "CorrErrorBack" : "Correction de l'erreur de syntaxe lors du retour en arrière",
+                "BlockHardware" : "Blocage des hardwares tests en mode examen",
```

Can you rename it too ?
```suggestion
                "BlockHardware" : "Blocage des hardwares tests en mode examen"
```